data/ka'het/ka'het missions.txt

```diff
@@ -99,56 +120,45 @@ mission "First Contact: Ka'het: Remnant 1B"
+		description `	As it was designed to be crewed as minimally as possible, only a tiny fraction of the station was inhabitable. The rest of the station, instead, was dedicated to advanced computing and targeting systems, most of which are still intact today.`
 	planet Osaeli
-		spaceport `For most of the thousands of years that passed between the time their discovery of spaceflight and their extinction, Osaeli was the de facto capital of the Builders, a key part of their economy and society. The first Het station was constructed orbiting this planet's star with the purpose of protecting them from any external threat.`
-		spaceport `	Incidentally, when the invasion broke out, Osaeli was also the world closest to the Ka'het fleets: it was bombarded and devastated for years until every Builder on the planet was killed.`
+		description `	For most of the thousands of years that passed between the time their discovery of spaceflight and their extinction, Osaeli was the de facto capital of the Builders, a key part of their economy and society. The first Het station was constructed orbiting this planet's star, with the purpose of protecting them from any external threat.`
 	planet Palarei
-		spaceport `Thanks to their earlier exploration vessels, this was where the Builders encountered the first lifeforms that evolved outside of their home planet. While the climate was drastically different for the Builders, the fruits the local flora produced were popular enough that Palarei became a major farming planet.`
-		spaceport `	The farms on Palarei were one of the few successful attempts at creating fully automatic facilities. Partly thanks to the rudimental repair systems they developed, a few dozen of them are still working even today.`
+		description `	Thanks to their earlier exploration vessels, this was where the Builders encountered the first lifeforms that evolved outside of their home planet. While the climate was drastically different for the Builders, the fruits the local flora produced were popular enough that Palarei became a major farming planet.`
 	planet Parii'het
-		spaceport `Parii'het was almost finished when the invasion broke out, and already capable of (slowly) creating Ka'het. Thanks to the mining operations on Alix and the occasional asteroid mining operated by the Mar drones, most of the facilities are still fully operational to this day.`
-		spaceport `	Different from most of the Graveyard, the station doesn't seem to have ever suffered any damage from the Ka'het, nor from the Builders trying to regain control over it.`
+		description `	Parii'het was almost finished when the invasion broke out, and already capable of creating Ka'het. Thanks to the mining operations on Alix and the occasional asteroid mining operated by the Mar unmanned vessels, most of the facilities are still fully operational to this day.`
```

Comment:
```suggestion
		description `	Parii'het was almost finished when the invasion broke out, and already capable of creating Ka'het. Thanks to the mining operations on Alix and the occasional asteroid mining by the Mar unmanned vessels, most of the facilities are still fully operational to this day.`
```
OR
```suggestion
		description `	Parii'het was almost finished when the invasion broke out, and already capable of creating Ka'het. Thanks to the mining on Alix and the occasional asteroid mining operated by the Mar unmanned vessels, most of the facilities are still fully operational to this day.`
```
Think I missed this the first pass. Just wanted to remove the double usage of `operation`. This is personal preference, and if you wanted to either keep as-is or substitute another word instead of removing one, that would work too.